src/features/user/ManageProjects/components/microComponent/SatelliteLayer.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+import { Source, Layer } from 'react-map-gl-v7/maplibre';
+
+const SatelliteLayer = () => {
+  const tiles = useMemo(
+    () => [
+      'https://wayback.maptiles.arcgis.com/arcgis/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
+    ],
+    []
+  );
+
+  return (
+    <Source
+      id="satellite-source"
+      type="raster"
+      tiles={tiles}
+      tileSize={256}
+      attribution="Esri, Maxar, Earthstar Geographics, and the GIS User Community"
```

Comment:
Update to 

"Historical imagery: © Esri, Maxar, Earthstar Geographics, and the GIS User Community (Esri Wayback)"